src/providers/slot-selector/slot-selector.ts

```diff
@@ -1,20 +1,24 @@
 import { Injectable } from '@angular/core';
 import { SlotItem } from './slot-item';
 import { TestSlotComponent } from '../../pages/journal/components/test-slot/test-slot';
+import { TestSlot } from '../../pages/journal/journal.model';
+import { JournalSlot } from '../../pages/journal/domain/JournalSlot';
 
 @Injectable()
 export class SlotSelectorProvider {
 
   constructor() { }
 
-  public getSlotTypes = (slots: any): SlotItem[] => {
+  public getSlotTypes = (journalSlots: JournalSlot[]): SlotItem[] => {
 
     const result: SlotItem[] = [];
-    if (Array.isArray(slots)) {
-      for (const slot of slots) {
+    if (Array.isArray(journalSlots)) {
```

Comment:
can we invert this if to reduce nesting?

Reply:
Discussed in person, not changing this one.